src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -649,9 +649,9 @@ private bool TryGetUnmanagedCallingConventionFromModOpt(MethodSignature signatur
                 }
             }
 
-            if (found && memberFunctionVariant)
+            if (memberFunctionVariant)
             {
-                callConv = GetMemberFunctionCallingConventionVariant(callConv);
+                callConv = GetMemberFunctionCallingConventionVariant(found ? callConv : (CorInfoCallConvExtension)PlatformDefaultUnmanagedCallingConvention());
```

Comment:
It seems the other place initializes `callConv` with `(CorInfoCallConvExtension)PlatformDefaultUnmanagedCallingConvention()`, is there a reason we don't do that here?

Reply:
I believe we want to identify the difference between specifying the default and not specifying a calling convention. This makes a difference for handling inlined P/Invoke stubs (since not specifying one counts as WinAPI)

Reply:
Ah okay. That is worth a comment as to why the default is different.